client/auth/index.js

```diff
@@ -10,8 +10,9 @@ import config from 'config';
 import controller from './controller';
 
 export default () => {
+	page( '/oauth-login', controller.oauthLogin );
```

Comment:
lets just comment here on the `page( '/oauth-login', controller.oauthLogin );` what it's doing and that it's not really calling `controller.oauthLogin` anytime, that's really not obvious.